### ISSUE TITLE
Tools

### ISSUE THREAD

**Issue by jonlab** [jonlab/NewAtlantis#4]
In New Atlantis, the visitor interacts with the other objects with "tools" that are standard and part of the Viewer. We have to define a tool list.

**Comment by jonlab** [comment]
Maybe the naming of "instrument" would be more related to Bacon's text ?

**Comment by rdrink** [comment]
Jonathan, I'm not sure what you mean here but I can tell you that when Ben and I worked on SynthEcology the avatars were very abstract but still needed a "manipulator" so that they could interact with objects. We made a very abstract "hand" and when we presented the piece in public people still had to ask what it was. However it was attached to a wand controller and so it moved as they moved their hand and they quickly figured it out. Unfortunately we are not using the same technology fo NA, so we need to think of something different.

**Comment by jonlab** [comment]
Yes, that is exactly that : the tools / instruments are a way to interact with the worlds and the other objects. this can be, for example, a stick (to hit other objects), the "push" action, a way to play sounds (like an acoustic test sound), a help set to the ear... I will put the current list on the Wiki.

**Comment by jonlab** [comment]
I just added a Wiki page "Tools, instruments" with a list (in-progress).

**Comment by rolandcahen** [comment]
Instruments is a good idea

**Comment by rolandcahen** [comment]
Here it is:
https://github.com/jonlab/NewAtlantis/wiki/Tools,-instruments

**Comment by jonlab** [comment]
Tools/instruments could be implemented could have a composite behaviour, for example : emit a sound AND throw an object.

**Comment by jonlab** [comment]
We could need special kind of tools that would be able to get and possibly use the standard analog inputs ("Move" joystick and "View" joystick) and inhibit the avatar default control. That would allow for more interaction schemes than just the simple "Action" trigger scheme.
